scripts/dataGenerator.js

```diff
@@ -1085,8 +1085,14 @@ db.initialize(function () {
                     process.exit(1);
                 }
 
-                calculateTotalNumberOfQuestions();
-                createAdmins();
+                db.removeAllFeedback(function(err, res) {
+                    if (err) {
+                        process.exit(1);
```

Comment:
there are 2 `process.exit(1)` in this file now, please organize it so that there is only one

Reply:
thers actually 4... i think its okay how it is, please confirm.